.travis.yml

```diff
@@ -16,21 +14,23 @@ jobs:
         - mix deps.get
         - mix test
     - name: "Deploy"
-      if: (NOT type IN (pull_request)) AND (branch = master)
       stage: deploy
-      env:
-        - MIX_ENV=prod
+      if: (NOT type IN (pull_request)) AND (branch = master)
       script:
-        - mix deps.get
-        - mix ecto.create
-        - mix ecto.migrate
+        - chmod +x ./install_cli_tools.sh
+        - ./install_cli_tools.sh
+        - cd terraform/
+        - ../terraform_cmd/terraform init
+        - chmod +x ./export_terraform_variables.sh
```

Comment:
FYI Git will track mode settings on files so if you changed the mode on the file locally, committed and pushed, you wouldn't need to change the mode in CI.

Reply:
That's good to know, I wasn't sure